ParkingLot/ParkingLot.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ParkingLotPlace
+{
+    public class ParkingLot
+    {
+        private Dictionary<string, string> ticket_Car = new Dictionary<string, string>();
+        private HashSet<string> usedTickets = new HashSet<string>();
+        private int capacity;
+        private int parkedCarsCount;
+
+        public ParkingLot(int capacity = 10)
+        {
+            this.capacity = capacity;
+            this.parkedCarsCount = 0;
+        }
+
+        public string ParkCar(string car)
+        {
+            if (ticket_Car.Count >= capacity)
+            {
+                throw new InvalidOperationException("No available position.");
+            }
+
+            var ticket = $"T{car}";
+            ticket_Car[ticket] = car;
+            parkedCarsCount++;
+            return ticket;
+        }
+
+        public string FetchCar(string ticket)
+        {
+            if (!ticket_Car.ContainsKey(ticket))
```

Comment:
WangKe:
In FetchCar method we could remove the ticket-car entry from ticket_car dictionary, then ticket_Car.ContainsKey(ticket) could be used for wrong ticket and used ticket checking as the used ticket be removed.